### ISSUE TITLE
The results not good as showed in the paper

### ISSUE THREAD

**Issue by LonglongaaaGo** [shepnerd/inpainting_gmcnn#60]
Hello ！ thanks for your repo！
Recently，I just use the PyTorch version to train the CelebA-HQ dataset with default hyper-parameter setting, and 24000 images for training, 6000 for testing. There are just different in dataset number setting which use 2000 images for testing in the paper. Finally, the results are shown below : 
psnr: 24.731138
ssim: 0.87873
I think, comparing with the results showed in the paper, the above results are not good. 
So, can you give me some advice to improve the performance? 
Thanks for your reply!

**Comment by LonglongaaaGo** [comment]
By the way, I think 24000 images are enough for training, so the scores for psne and ssim would not have a large gap between them.

**Comment by shepnerd** [comment]
We did not evaluate the quantitative performance of the released PyTorch version. Its quantitative results may differ from that in the paper since it adopts some different implementations (see disclaims in readme.md). 
To improve PSNR and SSIM, you can just increase the reconstruction loss weight and train the model a little longer. Further minimizing the pixel-wise differences between the prediction and ground truth will definitely raise PSNR and SSIM.

**Comment by LonglongaaaGo** [comment]
thank you, I would try!
